ksp_plugin.py

```diff
@@ -438,40 +441,82 @@ def unique(self, seq):
                 seen.add(item)
                 yield item
 
+    def on_load_async(self, view):
+        if sublime_version >= 4000:
+            global picture_filenames_compl, import_filenames_compl
+            workspace_files = sublime.active_window().folders()
+            script_path = sublime.active_window().active_view().file_name()
+            for folders in workspace_files:
+                img_list = Path(folders).rglob("*.png")
```

Comment:
TGA is also a supported file format, and so is SVG and WEBP for Native UI, just FYI.